### DISCUSSION TITLE
i18nAlly support

### DISCUSSION THREAD

**Discussion by mkbctrl** [vinissimus/next-translate#902] [Q&A]
Hey,

did anyone successfully setup the VScode i18n Ally extension to work with `next-translate`?
Usually it does work out of the box, the only thing that I frequently change is the path to locale files. In `next-translate` case I am unable to make it work.

I have the latest version of both VScode (1.70.2) and the extension (2.8.1).

Here's my example component:

![image](https://user-images.githubusercontent.com/10107362/186894866-26b03f60-b248-4dca-b0ff-42bf8c209af9.png)
![Screenshot 2022-08-26 at 13 42 23](https://user-images.githubusercontent.com/10107362/186896025-4560642f-be11-4a78-bb3d-e85db973433a.png)

locales file `auth.json`:
```
{
  "header": {
    "title": "Create Your Free Account",
    "subTitle": "Already have an account?",
    "link": "Sign in"
  },
  "divider": { "text": "or" },
  "policy": {
    "text": "By signing up you agree to our",
    "link": "privacy policy"
  }
}
```

The settings I changed are the following:
```
 "i18n-ally.localesPaths": ["locales"],
  "i18n-ally.extract.keyPrefix": "{fileName}", // so I assume when I type: "header.title" the extension should make it "auth.header.title" or am I dumb?
```

folder structure:
![Screenshot 2022-08-26 at 13 38 38](https://user-images.githubusercontent.com/10107362/186895437-23b3cb72-a39a-436c-8166-c1385f96ddbc.png)

config:
```
{
  "locales": ["en"],
  "defaultLocale": "en",
  "pages": {
    "/auth/sign-up": ["auth"]
  }
}

```

Anyone had any issues with that extension in the past? Did I miss smth?

The only case when the autocompletion works is when I type the full path:

https://user-images.githubusercontent.com/10107362/186896388-4148629a-bf02-478a-806d-f1c038d3e309.mov

![Screenshot 2022-08-26 at 13 46 05](https://user-images.githubusercontent.com/10107362/186896585-c299a55e-a31e-406a-af96-c8d042b801ed.png)

but it doesn't render the text but the string path (auth.header.link) in my frontend app...

**Comment by aralroca** [comment]
This is not supported but would be interesting to do it in the feature

**Comment by cassus** [comment]
See related issue:
- https://github.com/aralroca/next-translate/issues/983
